### PR TITLE
18 ionic get crawls

### PR DESCRIPTION
**Pull request by hkp108**
closes 18

### CODE REVIEW

File: test/e2e/crawlsFeature.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('Seeing all Crawls created', function() {
+  it('has a list of crawls', function() {
+    browser.get('/#/app/crawls');
+    expect(browser.getTitle()).toEqual('Crawls');
+  });
```
Comment (olmesm) [inline test/e2e/crawlsFeature.js]: can we create a crawl then expect it?

File: www/index.html [hunk 2]
```diff
@@ -16,6 +16,10 @@
     <script src="js/controllers/AppCtrl.js"></script>
     <script src="js/controllers/PlaylistsCtrl.js"></script>
```
Comment (hawksdoves) [inline www/index.html]: can we remove these lines (17+18) as they are referring to the PlaylistCtrl?

File: www/js/controllers/CrawlsController.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+  var self = this;
+
+  self.crawls = [{name: "haha"}];
+
+  self.getCrawls = function(){
+    CrawlService.getCrawls().then(
+      function(data){
+        self.crawls = data;
+      });
```
Comment (hawksdoves) [inline www/js/controllers/CrawlsController.js]: can we refactor: replacing lines 7-10 with 
CrawlService.getCrawls().then(
  function(data){
    self.crawls = data
  });

I.e. why do we need to wrap that code inside of a function, when its soul purpose seems to be to set self.crawls?

**Comment by hawksdoves** [comment]
🍻

**Pull request merged**